File: app.prueba/src/main/kotlin/com/example/app/prueba/entity/Activity.java
import jakarta.persistence.Entity;
import com.example.app.prueba.entity.Activity
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Table;
import org.springframework.data.annotation.Id;

import java.time.LocalDateTime;

@Entity
@Table(name = "activities")
public class Activity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String type;
    private String description;
    private Long leadId;
    private LocalDateTime timestamp;

    // Getters and Setters
}

File: app.prueba/src/main/kotlin/com/example/app/prueba/entity/Activity.kt
import jakarta.persistence.*
import java.time.LocalDateTime

@Entity
@Table(name = "activities")
class Activity {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private val id: Long? = null

    @ManyToOne
    @JoinColumn(name = "lead_id", nullable = false)
    private val lead: Lead? = null

    @Column(nullable = false)
    private var description: String? = null

    @Column(name = "date_time", nullable = false)
    private var dateTime: LocalDateTime? = null // Getters and setters
}